### PR TITLE
Handle None values in DBInserter

### PR DESCRIPTION
**Pull request by robertdstein**
`np.unique` cannot deal with `None`, so swap to `pd.unique`.

Also implement weighting for ra/dec in WINTER source table. 

Also also add WINTER configuration for `full_subset`, which is just `full` but for one board.

### CODE REVIEW

File: mirar/pipelines/winter/models/_candidates.py [hunk 1]
```diff
@@ -56,6 +58,7 @@ class CandidatesTable(WinterBase):  # pylint: disable=too-few-public-methods
     objectid = Column(VARCHAR(40), nullable=False, unique=False)
     deprecated = Column(Boolean, nullable=False, default=False)
     jd = Column(Float, nullable=False)
+    utctime = Column(DateTime(timezone=True))
```
Comment (virajkaram) [inline mirar/pipelines/winter/models/_candidates.py:61]: Why is this required? Isn't jd sufficient?

Comment (robertdstein) [reply mirar/pipelines/winter/models/_candidates.py:61]: This will enable to do easy native time queries using psql e.g:

```
select count(candidid) from candidates where AGE(CURRENT_TIMESTAMP, utctime) < '48 hours';
```

Additionally we can monitor candidates in the dashboard

Comment (virajkaram) [reply mirar/pipelines/winter/models/_candidates.py:61]: OK

**Review by virajkaram — APPROVED** [review]
